src/sink/rest.rs

```diff
@@ -391,6 +393,27 @@ impl Sink for Rest {
             Ok(None)
         } else {
             error!("Dropped data due to overload");
+            let mut error_ids = EventIdGenerator::new(sink_uid);
```

Comment:
can you put this into the struct? if used this way, every error event will have the same id.

Reply:
oh yes that makes sense, thanks!